### ISSUE TITLE
Add support for Google Drive shortcuts

### ISSUE THREAD

**Issue by aetonsi** [PhilippC/keepass2android#1955]
Hello,
first of all, thank you for re-enabling Google Drive support recently.
I have a feature request. I recently discovered that you can create shortcuts in Google Drive. You can do that by right clicking an item (folder or file) and selecting the following option:
![image](https://user-images.githubusercontent.com/18366087/160280115-b039520a-7f7b-4ab0-b98e-6d5fcaf5a158.png)

---

You can do that, for example, with files that are "shared with me". And that's what i did with my keepass folder: i shared it with my Google work account, and then i created a shortcut from "shared with me" to the root of my work account.
This is the result on gdrive desktop:
![image](https://user-images.githubusercontent.com/18366087/160280066-65b3916a-2205-4d20-8495-4680a120ead2.png)

---

The problem is, in K2A these "shortcuts" are seen as a 0 byte file. If you try to open it in the file picker, K2A opens it as a file, and it obviously doesn't work. This is what you see in K2A:
![image](https://user-images.githubusercontent.com/18366087/160280368-785860e1-e4ce-42e2-ba70-c7239b7a1fd4.png)

---

So i'm asking you if it would be possible to add support for these shortcuts.
Ask me anything if you need more explanation.

Thank you for your time

**Comment by aetonsi** [comment]
Also see this Google article about drive shortcuts: [https://support.google.com/drive/answer/10864219](https://support.google.com/drive/answer/10864219)

> The replacement process will start in 2022. You’ll get a notification in Google Drive a few weeks before your replacement starts.

**Comment by erichiller** [comment]
Yes  same here, I use shortcuts to share the DB with my wife and this doesn't work with kp2a.

**Comment by a-hackw** [comment]
Yes, I share a database with my wife as well. It worked until recently when google replaced her reference to the file with a shortcut.

**Comment by akiross** [comment]
Same issue here. Maybe, instead of supporting links, another option could be to be able to navigate into shared directories as well.

**Comment by leonjanzen-work** [comment]
Like others, I also share a file with my wife, and since Google changed the shared files to shortcuts, the sharing no longer works. If it's possible to get this app to open the Google Drive file shortcuts, it would be great to have the sharing working again!

**Comment by leonjanzen-work** [comment]
I just discovered that on Android, instead of choosing the "Google Drive" option, I can select the "System file picker", which includes access to my Google Drive account, and through that route, I can open a shortcut to a file, and it works just fine.
![Screenshot_20221123-153953](https://user-images.githubusercontent.com/6565814/203658318-c4cbf2c2-d8d9-4edf-9715-e890cdbca660.png)

**Comment by Trinition** [comment]
I can confirm that the "System File Picker" approach works for me, both in loading the file, and in saving changes to it.  Now I need to switch my spouse's device over to it.
